packages/gatsby-source-shopify/src/query-builders/product-variants-query.ts

```diff
@@ -38,7 +42,9 @@ export class ProductVariantsQuery extends BulkQuery {
                       width
                       originalSrc
                       transformedSrc
-                    }
+                    }${
```

Comment:
To make this a bit cleaner and more gql-like the [@skip directive](https://dgraph.io/docs/graphql/queries/skip-include/) could be used. I don't think it's a big deal here though.

Reply:
Good call, i change it to use `@include` and it's much cleaner.